go.mod

```diff
@@ -4,8 +4,8 @@ go 1.18
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
-	github.com/ava-labs/avalanche-network-runner v1.2.4-0.20221006162609-e7eb33c1e830
```

Comment:
not sure if this is a problem or not

Reply:
Will merge locally and close this, thanks for putting this up 🙌